guides/TERMINOLOGY.md

```diff
@@ -0,0 +1,7 @@
+#Terminology
+- Product - A piece of software controlled by an operator (e.g. RHSSO, 3scale, etc)
+- Resource - Any CRD that is managed by the product's operator. 
```

Comment:
IMHO: resource isn't necessarily a CRD, it can reference k8s/OpenShift native objects/resources :)

Reply:
removed this line as my resources definition essentially matches a kubernetes resources definition now.